### PR TITLE
Expo Playground Demo

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
# Motivation

To give potential users of our library a better understanding of the look and feel on real devices having the current playground always up to date on expo can provide a lot of benefit. As the expo version uses the normal RN playground as dependency there is no "magic" that might break in the future.

# Test plan

You can cd into the expo playground example directory, `npm install` and try the expo commands. (You need to install the expo CLI before that though)

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (skevy) [inline circle.yml]: Looks like you have a merge conflict here.

File: website/src/App.js [hunk 2]
```diff
@@ -43,13 +43,24 @@ type DocPageConfig = {
 };
 
 const createDocPage = (
+<<<<<<< HEAD
```
Comment (skevy) [inline website/src/App.js]: Merge conflict...

File: website/src/AppFrame.js [hunk 3]
```diff
@@ -51,9 +55,13 @@ class AppFrame extends React.Component {
       state: navigation.state.routes[navigation.state.index],
     });
     const { state } = navigation;
+<<<<<<< HEAD
```
Comment (skevy) [inline website/src/AppFrame.js]: Merge conflict...

File: circle.yml (hunk 1 above)
Comment (skevy) [inline circle.yml]: Looks like you missed it here @DanielMSchmidt.

**Comment by skevy** [comment]
One more thing @DanielMSchmidt -- you need to run eslint --fix on the files you changed, so that they're prettiered.

**Comment by DanielMSchmidt** [comment]
Sure, will do so later

**Comment by DanielMSchmidt** [comment]
@skevy Ran prettier, can we add running the format command in the readme, as pre-commit hook, and in the CI maybe? Could do a PR if you like 👍

**Comment by skevy** [comment]
@DanielMSchmidt yah -- we should definitely do this.

We shouldn't run Prettier FOR you, but we should run eslint pre-commit and recommend that you run it.

We can use Husky + Lint-Staged for this -- maybe you can put together a PR?

**Comment by MrHubble** [comment]
@DanielMSchmidt @skevy when I scan the QR code on my phone I get "no experience found": 

file_000

**Pull request merged**